src/containers/profile-page/ProfilePageProvider.tsx

```diff
@@ -620,7 +620,7 @@ class ProfilePage extends PureComponent<ProfilePageProps, ProfilePageState> {
 
   getIsArtist = () => {
     const { profile } = this.props.profile
-    return !!profile && profile.is_creator && profile.track_count > 0
+    return !!profile && profile.track_count > 0
```

Comment:
FYI, we haven't really settled on a convention for this but we can also write profile?.track_count > 0 ?? false if you prefer

Reply:
Nice!
I think we have a nice opportunity here to discuss some coding style. I'm usually open to most things. We can decide whether we want to converge to one style gradually, or diverge. Less headache with the former, more fun with the latter. Let's discuss!